spec/controllers/events_controller_spec.rb

```diff
@@ -105,7 +105,7 @@
       let(:params) { FactoryGirl.attributes_for(:event) }
 
       it 'throws a not authorized error' do
-        expect { subject }.to raise_error(Pundit::NotAuthorizedError)
+        expect(subject).to redirect_to forbidden_path
```

Comment:
You could leave these specs in and write more for the redirect. We are still raising the error but now we are rescuing it.